cmd/trade.go

```diff
@@ -331,14 +333,15 @@ func makeStrategy(
 	options inputs,
 	threadTracker *multithreading.ThreadTracker,
 	db *sql.DB,
+	tracker *metrics.Tracker,
```

Comment:
let's call this `metricsTracker` since this is our third type of tracker in the codebase

Reply:
Sounds good. Reasoning here is because the Go best practice is to avoid stuttering (my linter was heavily complaining with this name), but I get the need to align the codebase.

Reply:
yeah, there's a lot of fixes we need to make with the codebase to make them more go-idiomatic. But I'm less concerned about that because we have a fair amount of standardization as it is and that makes it easy(-ier) to make such changes.